client/src/components/UsedOnTable/UsedOnTable.js

```diff
@@ -64,23 +63,24 @@ class UsedOnTable extends PureComponent {
       id,
       title,
       type,
-      state,
       link,
     } = data;
 
-    const badge = (state)
-      ? <span className={classnames('badge', 'used-on__badge', `status-${state}`)}>{state}</span>
-      : null;
-
     const titleLabel = (link)
-      ? <a className="used-on__edit-link" href={link}>{title} {badge}</a>
-      : <span>{title} {badge}</span>;
+      ? <a className="used-on__edit-link" href={link}>{title}</a>
+      : <span>{title}</span>;
+
+    const handleTrClick = (event) => {
+      event.target.querySelector('a').click();
```

Comment:
```suggestion
      window.location = link
```

I assume this is the intention of the TR click. I don't think using DOM query API to manually trigger the click is very obvious, and probably more prone to bugs.